lnd.go

```diff
@@ -268,6 +268,9 @@ func Main(cfg *Config, lisCfg ListenerCfg, implCfg *ImplementationCfg,
 
 	rpcServerOpts := interceptorChain.CreateServerOpts()
 	serverOpts = append(serverOpts, rpcServerOpts...)
+	serverOpts = append(serverOpts, []grpc.ServerOption{
+		grpc.MaxRecvMsgSize(200 * 1024 * 1024),
```

Comment:
This number is now used twice in this file. Add it as a constant to the `lnrpc` package? Bonus points if we also use that constant in `lncli/main.go`.

Reply:
Good point, done!